R-package/R/utils.R

```diff
@@ -6,13 +6,8 @@ lgb.is.Dataset <- function(x) {
   lgb.check.r6.class(x, "lgb.Dataset") # Checking if it is of class lgb.Dataset or not
 }
 
-# use 64bit data to store address
-lgb.new.handle <- function() {
-  0.0 # Return numeric type in R
```

Comment:
When using C/C++ in R, it is possible to read/write memory of R. It is possible LightGBM writes in R memory when it should not.